trunner/config.py

```diff
@@ -162,7 +162,7 @@ def _parse_shell_command(self, config: dict):
 
             binary = binary_path.name
             path = binary_path.parent if binary_path.is_absolute() else self.ctx.target.exec_dir()
-            prefix = str(path) + "/" if self.ctx.target.rootfs else "sysexec "
+            prefix = "" if self.ctx.target.rootfs else "sysexec "
```

Comment:
Does not work for the target host-generic-pc
 
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/f9500c4d-cbab-4146-a4e6-c8899d122b1a)

Cause, in this way, the path to the test is currently being provided for host-generic-pc:
![image](https://github.com/phoenix-rtos/phoenix-rtos-tests/assets/116297248/519da096-fc1f-4d56-b629-9cb4639bf153)